mobile/library/swift/grpc/GRPCStream.swift

```diff
@@ -61,11 +61,16 @@ public final class GRPCStream: NSObject {
     return self
   }
 
-  /// Close this connection.
+  /// Close the stream, wait for the peer to finish before actually closing the stream.
```

Comment:
nit: I notice that these have "///" instead of "//". Is that a Swift thing or a bug?

Reply:
Swift thing/our convention: `///` for docs, `//` for comments